### ISSUE TITLE
Gold items wont work

### ISSUE THREAD

**Issue by shaarus** [ThinkInvis/RoR2-ClassicItems#84]
First time posting on github, but the item where it passively gives you gold and the item which gives u 25% more gold isnt working at all

**Comment by cheeeeeeeeeen** [comment]
@shaarus Please be more clear.

**Comment by Theray070696** [comment]
Can confirm this happens for a couple characters, notably Artificer, unsure of others atm. Affected items are Life Savings (not giving money at all), and possibly Smart Shopper (haven't personally tested, but that's what OP is referring to. I'll see if I can grab some logs today.

**Comment by imakunee** [comment]
Are you using ShareSuite anyone? That seems to cause an issue with gold items from multiple mods as there needs to be specific implementations for it. @Theray070696 @shaarus I noticed this when trying to figure out why The House wasn't getting gold from these items.

**Comment by ThinkInvis** [comment]
Once I have my mods updated for the latest patches, I won't be able to test this unless I have *at the very least* a mod list; a full console log would be greatly preferred. The issue template is there for a reason.

**Comment by MrNyoom** [comment]
[LogOutput.log](https://github.com/ThinkInvis/RoR2-ClassicItems/files/6425321/LogOutput.log)
After some testing I found that the bug only happens on modded survivors. I tested both life savings and smart shopper. Smart shopper worked 100% of the time and life savings does not work at all. I noticed the gold bug happen with another mod that gave gold ([broken blood tester from Starstorm](https://starstormmod.fandom.com/wiki/Broken_Blood_Tester)). It seems to be a compatibility issue with the modded survivors and the items. 
The mods I used were item stats mod, Paladin mod, classic items, and all the dependencies for said mods.
